ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/datacolumns/DataColumnSidecarCustody.java

```diff
@@ -29,4 +32,6 @@ SafeFuture<Optional<DataColumnSidecar>> getCustodyDataColumnSidecar(
   SafeFuture<Void> onNewValidatedDataColumnSidecar(DataColumnSidecar dataColumnSidecar);
 
   AsyncStream<DataColumnSlotAndIdentifier> retrieveMissingColumns();
+
+  List<UInt64> getCustodyColumnIndices(UInt64 epoch);
```

Comment:
Doesn't this depend on spec only? Do we really need this method in this interface?

Reply:
Oh, I see, you want to get current custody group count from the implementing class. 
But from my perspective it is not the responsibility of `DataColumnSidecarCustody`.
I would actually think of decoupling it from here